Networking/Sources/Networking/Models/Products/Product.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  File.swift
+//  
+//
+//  Created by Omar Ahmed on 23/06/2022.
+//
+
+import Foundation
+
+public struct Product: Codable {
```

Comment:
```suggestion
public struct Product: Decodable {
```
Since you're going to consume/ decode it only (You're not going to send it back)